provider/resources.go

```diff
@@ -421,20 +424,24 @@ func preConfigureCallback(vars resource.PropertyMap, c shim.ResourceConfig) erro
 // managedByPulumi is a default used for some managed resources, in the absence of something more meaningful.
 var managedByPulumi = &tfbridge.DefaultInfo{Value: "Managed by Pulumi"}
 
+//go:embed cmd/pulumi-resource-aws/bridge-metadata.json
+var metadata []byte
```

Comment:
Ah so this is how the PF provider gets its renames? Interesting. Sounds like it doesn't really scale easily to N but it's a limitation that tis fine for now!